src/newcharacter.cpp

```diff
@@ -487,7 +488,7 @@ int player::create(character_type type, std::string tempname)
 
     // Likewise, the asthmatic start with their medication.
     if (has_trait("ASTHMA")) {
-        tmp = item("inhaler", 0, false);
+        tmp = item( "inhaler", 0, item::find_type( "inhaler" )->charges_default() );
```

Comment:
Uh, that looks ugly. And it breaks encapsulation of the `itype`.

How about using a tag as parameter:
```C++
struct default_charges_tag {
};
class item {
    item(const itype_id &id, int turn, default_charges_tag);
};

// and in the cpp file:
item::item(const itype_id &id, int turn, default_charges_tag)
: item(id, turn, item::find_type( "inhaler" )->charges_default() ) );

// used like this:
item tmp = item( "inhaler", 0, default_charges_tag{} );
```

Or maybe add a named constructor.